src/apb/engine.py

```diff
@@ -623,20 +623,11 @@ def cmdrun_init(**kwargs):
     else:
```

Comment:
At line 619, we split on `/` with:

```
     apb_tag_arr = kwargs['tag'].split('/')
     apb_name = apb_tag_arr[-1]
     if apb_name.lower().endswith("-apb"):
         app_name = apb_name[:-4]
```

As it stands, with these changes, a `apb init <my_org>/my_apb` would ignore the `<my_org>/`. In order to build the image with the organization  like so `apb build --tag <my_org>/my_apb` or `apb build --registry docker.io/<my_org>` for the desired result. Just wanted to point this out to get feedback.

Reply:
Does it make sense here to add org as an option to `apb build`? I would expect to be able to include my org when building using the tooling but --registry docker.io/<org> isn't that intuitive.